app/src/main/java/com/tari/android/wallet/ui/common/CommonActivity.kt

```diff
@@ -146,7 +146,7 @@ abstract class CommonActivity<Binding : ViewBinding, VM : CommonViewModel> : App
 
     override fun onStop() {
         shakeDetector.stop()
-        dialogManager.dismiss()
+        dialogManager.dismiss() // TODO don't dismiss dialogs onStop. Think about a better way to handle this.
```

Comment:
Todo?

Reply:
@TruszczynskiA Yes, sometimes I see weak places in the code and try not to forget about them at least by using TODOs